### ISSUE TITLE
Create new hematodinium transcriptome

### ISSUE THREAD

**Issue by sr320** [RobertsLab/resources#1136]
Take all reads do the blast / megan pipeline pulling our all Alveolata reads 
assemble

**Comment by kubu4** [comment]
Hmmm, already done!

https://robertslab.github.io/sams-notebook/2020/08/14/Transcriptome-Assembly-Hematodinium-Transcriptomes-v1.6-and-v1.7-with-Trinity-on-Mox.html

Will update Genomic Resources readthedocs ASAP (no later than Monday).

**Comment by kubu4** [comment]
GAAAAAAAAH!!!

While documenting this on our readthedocs, I discovered a _critical_ copy/paste error in those assemblies (which, in turn, affects all downstream annotation, too). For the curious, I had copied/pasted SBATCH script info from _C.bairdi_ transcriptome assemblies and neglected to change the location of the FastQ files! So, the "_Hematodinium_" transcriptomes were generated with _C.bairdi_ reads!

New transcriptome assembly job is in queue on Mox. Will report back when complete.

Sorry!
